test/Interop/Cxx/static/constexpr-static-member-var-errors.swift

```diff
@@ -0,0 +1,10 @@
+// RUN: %target-swift-ide-test -print-module -module-to-print=ConstexprStaticMemberVarErrors -I %S/Inputs -source-filename=x -enable-cxx-interop 2>&1 | %FileCheck %s
+
+// Clang does not properly mark decls as invalid on Windows, so we won't get an
+// error here.
+// XFAIL: OS=windows-msvc
```

Comment:
While I added a special-case so that we no longer crash on Windows, I still can't enable this test, because Clang won't generate an error on Windows. @varungandhi-apple do you think it would be a good idea for me to add an additional test that just ensures we don't crash when importing `ConstexprStaticMemberVarErrors` (that would run on Windows)?